### ISSUE TITLE
Missing purescript-generics-rep

### ISSUE THREAD

**Issue by digy** [thomashoneyman/purescript-halogen-formless#38]
## Environment

* `"purescript-halogen": "^4.0.0",`
* `"purescript-halogen-formless": "^0.4.0",`

## Current behavior
Installed purescript-halogen-formless with bower.
When building getting this error:
```
Error 1 of 2:

  in module Formless.Types.Component
  at bower_components/purescript-halogen-formless/src/Formless/Types/Component.purs line 7, column 1 - line 7, column 40

    Module Data.Generic.Rep was not found.
    Make sure the source file exists, and that it has been provided as an input to the compiler.


  See https://github.com/purescript/documentation/blob/master/errors/ModuleNotFound.md for more information,
  or to contribute content related to this error.

Error 2 of 2:

  in module Formless.Types.Component
  at bower_components/purescript-halogen-formless/src/Formless/Types/Component.purs line 8, column 1 - line 8, column 43

    Module Data.Generic.Rep.Show was not found.
    Make sure the source file exists, and that it has been provided as an input to the compiler.


  See https://github.com/purescript/documentation/blob/master/errors/ModuleNotFound.md for more information,
  or to contribute content related to this error.
```
After manually installing `purescript-generics-rep` the build succeeds.

## Expected behavior

Maybe `purescript-generic-rep` dependency should be added of the `purescript-halogen-formless` ?

**Comment by thomashoneyman** [comment]
You're right! I tested dependencies by doing `pulp init` in an empty directory and then bower installing only Formless, and that builds just fine -- but that's because Pulp includes some other dependencies by default, like `effect`, `console`, `prelude`, and so on. When I cloned this in a project with _only_ Formless as the dependency I see the same error.

Looks like adding the generics-rep dependency does fix things.
